setup.py

```diff
@@ -29,7 +29,10 @@
         "flake8-breakpoint>=1.1.0,<2",  # detect breakpoints left in code
         "flake8-print>=4.0.0,<5",  # detect print statements left in code
         "isort>=5.10.1,<6",  # Import sorting linter
-        "pandas-stubs==1.2.0.62",  # NOTE: Needed due to mypy types
```

Comment:
This is no longer necessary?

Reply:
This dependency is listed twice. You can see it above by the other `mypy` related plugins.